examples/stable_diffusion_v2/dreambooth_finetune.md

```diff
@@ -189,6 +189,38 @@ python train_dreambooth.py \
     --with_prior_preservation=False \
 ```
 
+#### 2.2.5 Training Command for DreamBooth with LoRA
+
+LoRA is a parameter-efficient finetuning method. Here, we combine DreamBooth with LoRA by injecting the LoRA parameters into the text encoder and the UNet of Text-to-Image model, and training with the prior preservation loss.
```

Comment:
during training? grammarly

Reply:
by injecting xxx, and training with xxx. It seems alright to me.

Reply:
gucci just checked with gpt :D